package.json

```diff
@@ -115,69 +115,87 @@
                 "title": "SFDX: Scan Current File for Performance Issues with ApexGuru"
             }
         ],
-        "configuration": {
-            "title": "Salesforce Code Analyzer",
-            "properties": {
-                "codeAnalyzer.pMD.customConfigFile": {
+        "configuration": [{
+			"title": "General",
+			"properties": {
+				"codeAnalyzerV5.enabled": {
```

Comment:
I'm comfortable including "v5" in the name of this setting, since it's not going to be a permanent setting.